### ISSUE TITLE
Incorrect Kalos Route Order

### ISSUE THREAD

**Issue by jk13pclick** [pokeclicker/pokeclicker#1410]
**Info:** Version: 0.8.2

**Description:** Kalos Route 22 is unlocked after clearing Kalos Route 3 and should be the 4th ordered route in Kalos. However, it is not and is instead the 23rd ordered Route in Kalos.

**Expected Behavior:** Kalos Route Order should go 1->2->3->22->4->5...etc.

**Additional Information:** Fixing this issue could make some Kalos Dungeons slightly unbalanced with respect to the Closest Route difficulty.

**Comment by Qwertypop4** [comment]
I agree it should not be do difficult, but wouldn't this puzzle bot?
